lib/rabbitmq/cli/ctl/commands/list_consumers_command.ex

```diff
@@ -29,7 +29,8 @@ defmodule RabbitMQ.CLI.Ctl.Commands.ListConsumersCommand do
   def info_keys(), do: @info_keys
 
   def merge_defaults([], opts) do
-    {Enum.map(@info_keys, &Atom.to_string/1), Map.merge(%{vhost: "/", table_headers: true}, opts)}
+    {Enum.map(@info_keys -- [:activity_status], &Atom.to_string/1),
```

Comment:
Should consumer activity status be a part of this PR?
Applies to the rest of the file.

Reply:
Good catch!